permissions/plugin-ant.yml

```
@@ -4,4 +4,5 @@ github: "jenkinsci/ant-plugin"
 paths:
 - "org/jenkins-ci/plugins/ant"
 developers:
-- "armfergom"
\ No newline at end of file
+- "armfergom"
+- "jglick"
```

@jglick I would be interested in be listed as developer 
```suggestion
- "jglick"
- "fcojfernandez"
```